portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,35 @@
   </head>
   <body>
```

Comment:
Inside head

I would update the title from "My portfolio" to Bibi's Portfolio. The stuff in the head tag is read by crawlers. I would also add a couple of other metadata tags to optimize SEO performance.

Example:
<meta name="description" content="Anthony Wang Front End Developer Full Stack Portfolio Software Engineer UI UX Design">
<meta name="keywords" content="Anthony Wang, Front, End, Developer, Full Stack, Portfolio, Software Engineer, UI UX Design,CSS, UCSD, Intuit, HTML, Designer, San Diego.">

Some more context:
https://www.wordstream.com/meta-tags
https://www.w3schools.com/tags/tag_meta.asp